projects/igniteui-angular/src/lib/combo/combo.component.ts

```diff
@@ -1274,6 +1287,10 @@ export class IgxComboComponent extends DisplayDensityBase implements IgxComboBas
      */
     public ngAfterViewInit() {
         this.filteredData = [...this.data];
+
+        if (this.ngControl) {
+            this._statusChanges$ = this.ngControl.statusChanges.subscribe(this.onStatusChanged.bind(this));
```

Comment:
Could this subscription be with a `takeUntil` pipe? It would still prevent memory leaks w/o the need of manually unsubbing in `ngOnDestroy`.
```
this._statusChanges$ = this.ngControl.statusChanges.pipe(takeUntil(this.destroy$)).subscribe(this.onStatusChanged.bind(this))
```